### ISSUE TITLE
things to fix

### ISSUE THREAD

**Issue by rheiland** [PhysiCell-Tools/PhysiCell-model-builder#30]
* setting Chemotaxis for cell type-signal-towards/against is not retained unless user does a Save

**Comment by drbergman** [comment]
colorbar for `.mat` coloring of cells seems to be located and sized by pixels, rather than normalized units sometimes causing it to enter the axes:
<img width="1840" alt="image" src="https://github.com/PhysiCell-Tools/PhysiCell-Studio/assets/22732264/6e5e79e2-c37a-4a4f-9bb2-fd212cce7be9">

**Comment by rheiland** [comment]
I noticed something similar a couple days ago which makes me think recent edits have caused it as it used to display properly. Not sure what may have caused it. One obvious debug is to test with previous releases.

**Comment by rheiland** [comment]
In this release, https://github.com/PhysiCell-Tools/PhysiCell-Studio/releases/tag/v2.32.0, we introduced colorbars for cells' discrete  scalars. Testing the release just before that (2.31.2), we don't see the weird placement of that lower colorbar. I imagine @vincent-noel is swamped with other things right now (as most of us are), but FYI. Positioning colorbars on resizable windows is tricky, but hopefully there's some "tight" mpl magic that will work.
<img width="794" alt="cbars1" src="https://github.com/PhysiCell-Tools/PhysiCell-Studio/assets/1774917/3aadf8b9-1551-48e6-826e-ea583f6fc2e4">
<img width="794" alt="cbars2" src="https://github.com/PhysiCell-Tools/PhysiCell-Studio/assets/1774917/bdbc7d28-bbf7-44b8-9dfb-6f5090255a83">

**Comment by rheiland** [comment]
and this screenshot shows the situation using the latest release, with discrete scalars colorbar essentiall going out-of-view on a window resize.
<img width="701" alt="cbars3" src="https://github.com/PhysiCell-Tools/PhysiCell-Studio/assets/1774917/9c50c235-2a29-4a4f-97ef-39e88f21415e">

**Comment by vincent-noel** [comment]
Yes, I noticed it too. It's on my todo list, but I couldn't find time yet...

**Comment by rheiland** [comment]
I've spent a LONG time looking at possible fixes for the colorbars and the discrete scalars legend. If anyone has time to test, it's in https://github.com/PhysiCell-Tools/PhysiCell-Studio/tree/dev-colorbars . I still need to test for physiboss vis.

**Comment by drbergman** [comment]
This fixes it for me on my m2 mac!

**Comment by vincent-noel** [comment]
Thanks Randy ! I'll have a look asap
